lib/tasks_private/terraform/placement_group/variables.tf

```diff
@@ -0,0 +1,86 @@
+variable "ibmcloud_api_key" {
+    description = "Denotes the IBM Cloud API key to use"
+    type = string
+}
+
+variable "ibmcloud_region" {
+    description = "Denotes which IBM Cloud region to connect to"
+    default     = "syd"
+}
+
+variable "vm_name" {
+    description = "Name of the VM"
+    type = string
+}
+
+variable "power_instance_id" {
+    description = "Power Virtual Server instance ID associated with your IBM Cloud account (note that this is NOT the API key)"
+    default =  "888f9bf2-ab12-48c4-9dc5-8da65de951a8"
```

Comment:
This isn't going to be true for someone with different credentials trying to run this right?  Looks like something that should be part of the config settings/secrets?